working-open.md

```diff
@@ -0,0 +1,46 @@
+# Working Open
+
+Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that we've developed a set of guidelines for how (and exactly what) and why we work in the open.
+
+## Why?
+
+- In order to share, provide opportunities for self- and collective- learning as we work alongside other new economy projects.
+- Not because a "default to open" is an ideal, but because it orients us toward negotiations a set of practices around our shared intentions.
```

Comment:
```suggestion
- Not because "default to open" is an ideal in itself, but because it orients us toward negotiating a set of practices around our shared intentions.
```